test/integration/targets/win_pester/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+local_test_files:
+  - "files/test01.test.ps1"
```

Comment:
No need to use quotes here.
For simplicity and consistency we always recommend examples to only quote where it is essential to do so.
Not just to avoid weird YAML problems, but also because it makes things easier to read, avoids nested quoting and so users can actually tell from the example when quotes are deliberate/necessary.

We have a long way to go, but every change is for the better :)

Reply:
>We have a long way to go, but every change is for the better :)

I'm learning a lot and this is great ;-)